examples/go/tagging/handler/handler.go

```diff
@@ -191,11 +194,11 @@ func validateConfig(cfg vcConfig) error {
 func debug() bool {
 	verbose := os.Getenv("write_debug")
 
-	if verbose == "false" {
-		return false
+	if verbose == "true" {
```

Comment:
nit: could be shortened to `if verbose {...}`
You might want to consider using a linter which highlights these.

Reply:
My linter had alerted me to these lines, but I ignored it because I didn't think the logic would work. 

```
S1008: should use 'return <expr>' instead of 'if <expr> { return <bool> }; return <bool>' (gosimple)
```

It suggested I do this (won't work):
```
return verbose
```

However, what you suggest will work.
```
if verbose {
   return true 
}

return false
```

Good eye @embano1 ! I'll go ahead and make that change.

Reply:
Ah, me complete #fail xD